subnet/kube/kube.go

```diff
@@ -138,7 +139,15 @@ func newKubeSubnetManager(c clientset.Interface, sc *subnet.Config, nodeName, pr
 	ksm.client = c
 	ksm.nodeName = nodeName
 	ksm.subnetConf = sc
-	ksm.events = make(chan subnet.Event, 5000)
+	scale := 5000
+	scaleStr := os.Getenv("K8S_SCALE")
+	if scaleStr != "" {
+		scale, _ := strconv.Atoi(scaleStr)
```

Comment:
We should be logging the error from Atoi?
Perhaps even return the error.

Reply:
updated